docker/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM centos:7
```

Comment:
I'm missing wihtin this file the following installation steps:

* libthrift-0.9.1 must overwrite the libthrift library given with Flume (I think it is 0.7.0).
* `cygnus-flume-ng` must replace `APACHE_FLUME_HOME/bin/flume-ng`. You will find it at https://github.com/telefonicaid/fiware-cygnus/blob/master/src/main/resources/cygnus-flume-ng